### PR TITLE
OCPBUGS-23432: Use shorter IP label for keepalived VIP

### PR DESCRIPTION
**Pull request by mkowalski**
This PR modifies the label used for an IP address assigned by keepalived to not include the name of the interface. This is because the maximum allowed lenght of the label is 15 characters, causing stuff to fail when interface itself has a name longer than 12 characters.

With this change the label will always be a simple string of length 3 and will not be susceptible to overflow for long NIC names.

Contributes-to: OCPBUGS-23432

### CODE REVIEW

**Review by yuqi-zhang — APPROVED** [review]

**Comment by cybertron** [comment]
/test e2e-metal-ipi

Seems fine to me. I assume the SDN side (which I think is why we added the label in the first place) is just looking for the "vip" substring?

**Comment by cybertron** [comment]
/hold

One of the on-prem jobs should pass before we merge this. Once that happens please feel free to remove the hold.

**Comment by mkowalski** [comment]
Yes, OVN-K8s is getting a respective fix via https://github.com/ovn-org/ovn-kubernetes/pull/4022

**Comment by mkowalski** [comment]
/unhold

e2e-metal-ipi passes

**Pull request merged**
